internal/pkg/gateway/server/mocks/mock_registry.go

```diff
@@ -0,0 +1,167 @@
+/*
+Copyright 2020 IBM All Rights Reserved.
```

Comment:
It's better to use auto-generated mocks. Search for "counterfeiter" or "mockery" (my favorite is the latter but most people in the project wrongly prefer the former) in the code base